pkgs/applications/gis/grass/default.nix

```diff
@@ -1,24 +1,25 @@
 { lib, stdenv, fetchFromGitHub, flex, bison, pkg-config, zlib, libtiff, libpng, fftw
-, cairo, readline, ffmpeg_3, makeWrapper, wxGTK30, netcdf, blas
-, proj, gdal, geos, sqlite, postgresql, libmysqlclient, python2Packages, libLAS, proj-datumgrid
+, cairo, readline, ffmpeg, makeWrapper, wxGTK30, netcdf, blas
+, proj, gdal, geos, sqlite, postgresql, libmysqlclient, python3Packages, libLAS, proj-datumgrid
+, zstd, pdal, wrapGAppsHook
 }:
 
 stdenv.mkDerivation rec {
   name = "grass";
-  version = "7.6.1";
+  version = "7.8.6";
 
   src = with lib; fetchFromGitHub {
     owner = "OSGeo";
     repo = "grass";
-    rev = "${name}_${replaceStrings ["."] ["_"] version}";
-    sha256 = "1amjk9rz7vw5ha7nyl5j2bfwj5if9w62nlwx5qbp1x7spldimlll";
+    rev = version;
+    sha256 = "sha256-zvZqFWuxNyA+hu+NMiRbQVdzzrQPsZrdGdfVB17+SbM=";
   };
 
   nativeBuildInputs = [ pkg-config ];
-  buildInputs = [ flex bison zlib proj gdal libtiff libpng fftw sqlite cairo proj
-  readline ffmpeg_3 makeWrapper wxGTK30 netcdf geos postgresql libmysqlclient blas
-  libLAS proj-datumgrid ]
-    ++ (with python2Packages; [ python python-dateutil wxPython30 numpy ]);
+  buildInputs = [ flex bison zlib proj gdal libtiff libpng fftw sqlite cairo
+  readline ffmpeg makeWrapper wxGTK30 netcdf geos postgresql libmysqlclient blas
+  libLAS proj-datumgrid zstd pdal wrapGAppsHook ]
+    ++ (with python3Packages; [ python python-dateutil wxPython_4_1 numpy ]);
```

Comment:
Can `wxPython_4_1` either be switched to `wxPython_4_0` or put with logic similar to this:
```
    ++ (with python3Packages; [ python python-dateutil numpy ] ++
                              lib.optional stdenv.isDarwin wxPython_4_0 ++
                              lib.optional stdenv.isLinux wxPython_4_1);
```

For reasons not entirely clear to me with #150908, `wxGTK30` builds on darwin x86, but `wxGTK31` fails with the error that `--enable-universal-binaries` (found [here](https://github.com/NixOS/nixpkgs/blob/c81e01e9b9ce248cfd15d28926fd286efe2374f4/pkgs/development/libraries/wxwidgets/3.1/default.nix#L90) isn't a valid option. I don't totally understand why, since it also exists in [3.0](https://github.com/NixOS/nixpkgs/blob/c81e01e9b9ce248cfd15d28926fd286efe2374f4/pkgs/development/libraries/wxwidgets/3.0/default.nix#L57).